README.md

```diff
@@ -43,93 +43,94 @@ pip install dash-tools
 
 ## **Usage Examples**
 
-Below are common usage examples. For a more in-depth tutorial on writing apps for Plotly Dash, see the [Plotly Dash Documentation](https://dash.plotly.com/layout). For information about dash-tools commands, read the [_Commands_](#commands) section.
+### A. Create an App
 
-### **Creating A New Project**
-
-> _For a more in-depth tutorial on using dash-tools, check out the [Create and Upload Your App](/docs/Configuring-Your-App.md) document!_
-
-Creating a new Dash project is very simple. The following command will create a new directory called "MyDashApp":
+1. Create a Dash project in a new directory called "MyDashApp" (using your terminal or command prompt):
 
 ```bash
 dash-tools --init MyDashApp
 ```
 
-Optionally, templates can be used. Check out [Templates](#templates) for more details.
-
-You can see what files are included with your new app:
+2. Open the default app:
```

Comment:
What does default app mean here?

Reply:
I changed it to "open the default app.py file that comes with this project". Feel free to tweak it even further to ensure it makes sense to you.